lib/elasticity/search.rb

```diff
@@ -216,21 +186,25 @@ def search
     end
 
     class ActiveRecordProxy
-      def self.from_hits(relation, hits)
-        ids = hits.map { |hit| hit["_id"] }
+      def self.from_hits(relation, search_definition, response)
```

Comment:
Since this method changed from taking an array of hits into taking more data, can we rename this to `map_response`?